Updates for schedule api changes (#1595)
temporalio/documentation: docs/tctl/schedule/describe.md

```shell
tctl schedule describe --sid 'your-schedule-id'
```

> The Schedule Spec in the output is canonicalized and might not be in the same form as it was input.

Reviewer: ```suggestion
Because the Schedule Spec is converted to canonical representations, the output might not be in the same form as it was input.
```
